Marlin/src/lcd/ultralcd.cpp

```diff
@@ -1481,6 +1481,9 @@ void MarlinUI::update() {
     #ifdef ACTION_ON_CANCEL
       host_action_cancel();
     #endif
+    #if ENABLED(HOST_PROMPT_SUPPORT)
+      host_prompt_open(PROMPT_INFO, PSTR("UI Aborted"), PSTR("Dismiss"));
+    #endif
```

Comment:
@foosel — I believe this is the one you objected to in our last discussion.

Reply:
Yup, the change is it now has a dismiss button. Id like some feedback on preferred direction to change these. I considered on G0/1 parsing if an info message is present dismiss it since itll be stale. That should resolve the final hesitation around the few mentioned in the description.

Reply:
I'm happy to proceed! It sounds like Dismiss button would be the default for all prompts not including a specific button.

Reply:
Except for the rare wait prompt (we only have one!).